index.html

```diff
@@ -1,20 +1,154 @@
+            <img class="hero-content-img" src="/images/fruit-three.jpeg" />
+          </div>
+        </div>
+        <div class="row hero-content-row">
+          <div class="sm-col-12 md-col-12 lg-col-4">
+            <img class="hero-content-img" src="/images/fruit-four.jpeg" />
+          </div>
+          <div class="sm-col-12 md-col-12 lg-col-4">
+            <img class="hero-content-img" src="/images/fruit-five.jpeg" />
+          </div>
+          <div class="sm-col-12 md-col-12 lg-col-4">
+            <img class="hero-content-img" src="/images/fruit-six.jpeg" />
+          </div>
+        </div>
+        <div class="row hero-content-row">
+          <div class="sm-col-12 md-col-12 lg-col-4">
+            <img class="hero-content-img" src="/images/fruit-seven.jpeg" />
+          </div>
+          <div class="sm-col-12 md-col-12 lg-col-4">
+            <img class="hero-content-img" src="/images/fruit-eight.jpeg" />
+          </div>
+          <div class="sm-col-12 md-col-12 lg-col-4">
+            <img class="hero-content-img" src="/images/kiwi.jpeg" />
+          </div>
+        </div>
+      </section>
+      <section class="container about">
+        <div class="row hero-content-row">
+          <div class="sm-col-12 md-col-12 lg-col-12">
+            <img
+              class="hero-content-img"
+              src="/images/fruit-vertical-three.jpeg"
+            />
+          </div>
+          <div class="sm-col-12 md-col-12 lg-col-12">
+            <h2 class="sm-col-12 md-col-12 lg-col-12">about</h2>
+            <p class="about-text">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit,
+              et maxime fuga officiis culpa nihil eligendi aperiam earum maiores
+              quod saepe quo perspiciatis Lorem ipsum dolor sit amet consectetur
+              adipisicing elit. Deserunt aspernatur nemo fugit repudiandae
+              deleniti ratione dolores perferendis vitae nostrum explicabo.
+              Consequuntur autem similique eos ducimus voluptatibus!
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="container contact">
+        <h2 class="sm-col-12 md-col-12 lg-col-12">contact</h2>
+        <div class=" sm-col-12 md-col-12 lg-col-12 contact-form">
+          <form class="contact-form-content">
+            <label for="name">Enter Your Name *</label><br />
+            <input type="text" id="name" placeholder="   Name" /><br />
+            <label for="email">Enter Your Email *</label><br />
+            <input type="text" id="email" placeholder="   Email" /><br />
+            <label for="phone">Enter Your Phone *</label><br />
+            <input type="text" id="phone" placeholder="   Phone" /><br />
+            <label for="message">Enter Your Message here *</label><br />
+            <input
+              class="message-box"
+              type="text"
+              id="message"
+              placeholder="   Message"
```

Comment:
If you want a multi-line input box, you can look into `<textarea>` instead of `<input>`.

Also, if you want to style a form element, use CSS - prefer `text-align` to spaces! There's even [a selector](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder) which allows you to change the style of the placeholder text.